backend/app/models/PlaneConfig.scala

```diff
@@ -0,0 +1,93 @@
+/*
+ *
+ * Lasius - Open source time tracker for teams
+ * Copyright (c) Tegonal Genossenschaft (https://tegonal.com)
+ *
+ * This file is part of Lasius.
+ *
+ * Lasius is free software: you can redistribute it and/or modify it under the
+ * terms of the GNU Affero General Public License as published by the Free
+ * Software Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Lasius is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Lasius. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package models
+
+import models.BaseFormat._
+import play.api.libs.json._
+import reactivemongo.api.bson.BSONObjectID
+
+import java.net.URL
+
+case class PlaneConfigId(value: BSONObjectID = BSONObjectID.generate())
+    extends BaseBSONObjectId
+
+case class PlaneSettings(checkFrequency: Long)
+
+case class PlaneTagConfiguration(useLabels: Boolean,
+                                 labelFilter: Set[String],
+                                 useMilestone: Boolean = false,
+                                 useTitle: Boolean = false)
+
+case class PlaneProjectSettings(planeProjectId: String,
```

Comment:
here we would need the workspace id as well

Reply:
Done